docs/docs/APIdocs/layers/nn/Initializers/BilinearFiller.md

```diff
@@ -0,0 +1,132 @@
+## BilinearFiller ##
+
+
+**Scala:**
+``` scala
+val initMethod = BilinearFiller
+
+```
+**Python:**
+```python
+init_method = BilinearFiller(bigdl_type="float")
```

Comment:
scala we didn't put [T] in constructor and we also don't expect user to use bigdl_type, right?